CMakeLists.txt

```diff
@@ -7,52 +7,86 @@ include(cotire)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 
-set(ENABLE_DEBUG_GRAPHICS TRUE)
-
-if (ENABLE_DEBUG_GRAPHICS)
-    add_definitions(-DRTT_CMAKE_ENABLE_DEBUG_GRAPHICS=1)
-else (ENABLE_DEBUG_GRAPHICS)
-    add_definitions(-DRTT_CMAKE_ENABLE_DEBUG_GRAPHICS=0)
-endif (ENABLE_DEBUG_GRAPHICS)
-
-
-set(CMAKE_CXX_FLAGS "\
---std=c++17 ${CMAKE_CXX_FLAGS} \
--Wall \
--Wno-unused-function \
--Wsign-compare \
--g \
-"
-        )
-
-
-
 
 add_library(roboteam_utils STATIC
         src/Polygon.cpp
+        include/roboteam_utils/Polygon.h
```

Comment:
is it necessary to put the .h files here as well? IIRC this should not be needed

Reply:
It's not necessary but consistent with the rest of our codebase. It also helps if we ever want to port (it does actually make a small difference).